Polyfills/Console/Source/Console.cpp

```diff
@@ -34,16 +34,82 @@ namespace
         }
     }
 
+    inline bool isNumberEnding(char c)
+    {
+        return c == 'd' || c == 'i' || c == 'f';
+    }
+
+    inline bool isStringOrObjectEnding(char c)
+    {
+        return c == 'o' || c == 'O' || c == 's';
+    }
```

Comment:
nit: I'm not sure this makes the code clearer. You already have to explain exactly what is happening when you call these functions and they are only called once. I think it's clearer without these functions.